starboard/xb1/shared/accessibility_get_caption_settings.cc

```diff
@@ -0,0 +1,194 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Windows.h>
+
+#include "starboard/common/memory.h"
+#include "starboard/shared/starboard/accessibility_internal.h"
+#include "starboard/xb1/shared/accessibility_extension.h"
+
+namespace starboard {
+namespace xb1 {
+namespace shared {
+namespace accessibility {
+
+namespace {
+
+using namespace Windows::Media::ClosedCaptioning;
+
+SbAccessibilityCaptionCharacterEdgeStyle
+ClosedCaptionEdgeEffectToSbAccessibilityCaptionCharacterEdgeStyle(
+    ClosedCaptionEdgeEffect edgeStyle) {
+  switch (edgeStyle) {
+    case ClosedCaptionEdgeEffect::Default:
+      return kSbAccessibilityCaptionCharacterEdgeStyleNone;
+    case ClosedCaptionEdgeEffect::None:
+      return kSbAccessibilityCaptionCharacterEdgeStyleNone;
+    case ClosedCaptionEdgeEffect::Raised:
+      return kSbAccessibilityCaptionCharacterEdgeStyleRaised;
+    case ClosedCaptionEdgeEffect::Depressed:
+      return kSbAccessibilityCaptionCharacterEdgeStyleDepressed;
+    case ClosedCaptionEdgeEffect::Uniform:
+      return kSbAccessibilityCaptionCharacterEdgeStyleUniform;
+    case ClosedCaptionEdgeEffect::DropShadow:
+      return kSbAccessibilityCaptionCharacterEdgeStyleDropShadow;
+  }
+}
+
+SbAccessibilityCaptionColor ClosedCaptionColorToSbAccessibilityCaptionColor(
+    ClosedCaptionColor color) {
+  switch (color) {
+    case ClosedCaptionColor::Default:
+      return kSbAccessibilityCaptionColorWhite;
+    case ClosedCaptionColor::White:
+      return kSbAccessibilityCaptionColorWhite;
+    case ClosedCaptionColor::Black:
+      return kSbAccessibilityCaptionColorBlack;
+    case ClosedCaptionColor::Red:
+      return kSbAccessibilityCaptionColorRed;
+    case ClosedCaptionColor::Green:
+      return kSbAccessibilityCaptionColorGreen;
+    case ClosedCaptionColor::Blue:
+      return kSbAccessibilityCaptionColorBlue;
+    case ClosedCaptionColor::Yellow:
+      return kSbAccessibilityCaptionColorYellow;
+    case ClosedCaptionColor::Magenta:
+      return kSbAccessibilityCaptionColorMagenta;
+    case ClosedCaptionColor::Cyan:
+      return kSbAccessibilityCaptionColorCyan;
+  }
+}
+
+SbAccessibilityCaptionFontFamily
+ClosedCaptionStyleToSbAccessibilityCaptionFontFamily(ClosedCaptionStyle style) {
+  switch (style) {
+    case ClosedCaptionStyle::Default:
+      return kSbAccessibilityCaptionFontFamilyCasual;
+    case ClosedCaptionStyle::MonospacedWithSerifs:
+      return kSbAccessibilityCaptionFontFamilyMonospaceSerif;
+    case ClosedCaptionStyle::ProportionalWithSerifs:
+      return kSbAccessibilityCaptionFontFamilyProportionalSerif;
+    case ClosedCaptionStyle::MonospacedWithoutSerifs:
+      return kSbAccessibilityCaptionFontFamilyMonospaceSansSerif;  // TODO: is it ok?
+    case ClosedCaptionStyle::ProportionalWithoutSerifs:
+      return kSbAccessibilityCaptionFontFamilyProportionalSansSerif;  // TODO: is it ok?
```

Comment:
how should it be mapped?
MonospacedWithSerifs->MonospaceSerif
MonospacedWithoutSerifs -> MonospaceSansSerif

Or:
how should it be mapped?
MonospacedWithSerifs->MonospaceSansSerif
MonospacedWithoutSerifs -> MonospaceSerif